hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt

```diff
@@ -6501,11 +7330,17 @@ e_hre = {
                 crimean_gothic = "Keleia" # Historical? Copied from Greek
                 croatian = "Celje"
```

Comment:
This one was "rauna" before for all south slavs. Source for "Celje"? Celje is in slovenia maybe misplaced name?

Reply:
The default name is "Cilli" and from what I can find on the wiki the slavic name is "Celje"